src/helm/benchmark/window_services/window_service_factory.py

```diff
@@ -62,7 +62,7 @@ def get_window_service(model_name: str, service: TokenizerService) -> WindowServ
             window_service = SantaCoderWindowService(service)
         elif model_name == "huggingface/gpt2":
             window_service = GPT2WindowService(service)
-        elif model_name == "together/bloom":
+        elif model_name == "together/bloom" or model_name == "together/bloomz":
             window_service = BloomWindowService(service)
```

Comment:
I don't think it's the same tokenizer: https://huggingface.co/bigscience/bloomz#cpu

Reply:
Added BLOOMZ tokenizer.

As far as I can tell, it's the exact same tokenizer, just under a different name.